6.2 -  Blacklist

c-paradis opened the pull request:
Hello,
I've added a new module to Opensips-cp.

This blacklist manager allows you to read, insert and edit entries in the database.
It checks if the admin has read-only or read-write permissions to allow him to add / edit entries.
With this tool you can also reload the global blacklist cache with an "apply changes" button.

bogdan-iancu commented:
Hi @c-paradis , let me review it shortly and come back with feedback. Thanks for the contributions, Bogdan

c-paradis commented:
Hi @bogdan-iancu , i've added another option to the blacklist manager.
At the begining i didn't think about having hundreds or thousands entries in the database... So now i've added a search functionality to filter results and find the desired entries faster than before.

It works for the globalblacklist and the userblacklist as well

bogdan-iancu commented:
Hi @c-paradis , the patch looks good, but before merging, please place the Copyright and License text in all the new files . See the example from the existing file, simply copy and replace the Copyright line with your name/business/etc.

bogdan-iancu commented:
Thank you @c-paradis , I will shortly review.
